pvr.vuplus/addon.xml.in

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.vuplus"
-  version="3.7.3"
+  version="3.7.4"
```

Comment:
As you have introduced a new feature, an update of the minor version is more appropriate => 3.8.0